arviz/plots/backends/matplotlib/compareplot.py

```diff
@@ -41,45 +43,50 @@ def plot_compare(
     if ax is None:
         _, ax = create_axes_grid(1, backend_kwargs=backend_kwargs)
 
-    if plot_ic_diff:
-        ax.set_yticks(yticks_pos)
-        ax.errorbar(
-            x=comp_df[information_criterion].iloc[1:],
-            y=yticks_pos[1::2],
-            xerr=comp_df.dse[1:],
-            color=plot_kwargs.get("color_dse", "grey"),
-            fmt=plot_kwargs.get("marker_dse", "^"),
-            mew=linewidth,
-            elinewidth=linewidth,
-        )
-
-    else:
-        ax.set_yticks(yticks_pos[::2])
-
     if plot_standard_error:
         ax.errorbar(
             x=comp_df[information_criterion],
             y=yticks_pos[::2],
             xerr=comp_df.se,
+            label="ELPD",
             color=plot_kwargs.get("color_ic", "k"),
             fmt=plot_kwargs.get("marker_ic", "o"),
-            mfc="None",
+            mfc=plot_kwargs.get("marker_fc", "white"),
             mew=linewidth,
             lw=linewidth,
         )
     else:
         ax.plot(
             comp_df[information_criterion],
             yticks_pos[::2],
+            label="ELPD",
             color=plot_kwargs.get("color_ic", "k"),
             marker=plot_kwargs.get("marker_ic", "o"),
-            mfc="None",
+            mfc=plot_kwargs.get("marker_fc", "white"),
             mew=linewidth,
             lw=0,
+            zorder=3,
         )
 
+    if plot_ic_diff:
+        ax.set_yticks(yticks_pos)
+        ax.errorbar(
+            x=comp_df[information_criterion].iloc[1:],
+            y=yticks_pos[1::2],
+            xerr=comp_df.dse[1:],
+            label="ELPD difference",
+            color=plot_kwargs.get("color_dse", "grey"),
+            fmt=plot_kwargs.get("marker_dse", "^"),
+            mew=linewidth,
+            elinewidth=linewidth,
+        )
+
+    else:
+        ax.set_yticks(yticks_pos[::2])
+
+    scale = comp_df[f"scale"][0]
+
     if insample_dev:
-        scale = comp_df[f"{information_criterion}_scale"][0]
         p_ic = comp_df[f"p_{information_criterion}"]
```

Comment:
This continues to be `p_loo` but now the fstring generates `p_elpd_loo` which is an invalid key. Not sure why tests are all passing, but it is the reason the doc related builds are failing.